### DISCUSSION TITLE
when i run the oh-my-zsh.sh ,error ocurr

### DISCUSSION THREAD

**Discussion by wonglay** [ohmyzsh/ohmyzsh#11064] [Help]
when i run the oh-my-zsh.sh ,error ocurr as follows:

```

Error: Oh My Zsh can't be loaded from: sh. You need to run zsh instead.
Here's the process tree:


 PPID   PID COMMAND
    1 17849 /Applications/iTerm.app/Contents/MacOS/iTerm2
17849 17850 /Users/wanglei/Library/Application Support/iTerm2/iTermServer-3.4.1
17850 59263 login -fp wanglei
59263 59264 -zsh
59264 59278 sh ./oh-my-zsh.sh
```

but i comfirm that i run it with zsh .

and zsh`s version is zsh 5.8.1 (x86_64-apple-darwin21.0)  when i run 'zsh --version'

**Comment by rwmitchell** [comment]
Process 59264 confirms that you are NOT running it with zsh, but sh - says so right there.

So, how ARE you running ./oh-my-zsh.sh ?

In your .zshrc file, which is where this probably is, you want to use:

```source $ZSH/oh-my-zsh.sh```

Equally, and this is for the OMZ maintainers, why DOES oh-my-zsh end in .sh and not .zsh ?
